frontend/packages/client/src/components/Community/CommunityPropsAndVoting.js

```diff
@@ -0,0 +1,177 @@
+import React, { useState } from "react";
+import { useVotingStrategies } from "hooks";
+import { AddButton, ActionButton } from "components";
+import { Bin } from "components/Svg";
+import { useModalContext } from "contexts/NotificationModal";
+import StrategyEditorModal from "./StrategyEditorModal";
+
+const StrategyInput = ({ index, commuVotStra, onDeleteStrategy } = {}) => {
+  return (
+    <div
+      key={`index-${index}`}
+      className="column is-12 is-mobile p-0 m-0 mb-4 fade-in"
+      style={{ position: "relative" }}
+    >
+      <input
+        type="text"
+        className="border-light rounded-sm p-3 column is-full"
+        defaultValue={commuVotStra}
+        style={{
+          width: "100%",
+        }}
+      />
+      <div
+        style={{
+          display: "flex",
+          justifyContent: "flex-end",
+          alignContent: "center",
+          position: "absolute",
+          right: 15,
+          top: 9,
+        }}
+      >
+        <div
+          className="cursor-pointer is-flex is-align-items-center"
+          onClick={() => onDeleteStrategy(index)}
+        >
+          <Bin />
+        </div>
+      </div>
+    </div>
+  );
+};
+
+export default function CommunityProposalsAndVoting({
+  communityVotingStrategies = [],
+  communityId,
+  updateCommunity,
+  updatingCommunity,
+} = {}) {
+  const { data: allVotingStrategies, loading: loadingAllStrategies } =
+    useVotingStrategies();
+
+  const [currentStrategies, setCurrentStrategies] = useState(
+    communityVotingStrategies.map((st) => ({ strategy: st, fromServer: true }))
```

Comment:
`fromServer: true` might not be needed, I want to test not to reload the component after the update